common/src/main/scala/com/gu/monitoring/CloudWatch.scala

```diff
@@ -0,0 +1,62 @@
+package com.gu.monitoring
+
+import java.util.concurrent.Future
+
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.regions.Regions.EU_WEST_1
+import com.amazonaws.services.cloudwatch.{AmazonCloudWatchAsync, AmazonCloudWatchAsyncClient, AmazonCloudWatchAsyncClientBuilder}
+import com.amazonaws.services.cloudwatch.model.{Dimension, MetricDatum, PutMetricDataRequest, PutMetricDataResult}
+import com.gu.aws.CredentialsProvider
+import com.gu.config.Configuration
+import com.gu.monitoring.CloudWatch.cloudwatch
+import com.typesafe.scalalogging.LazyLogging
+
+class CloudWatch(metrics: Seq[Dimension]) extends LazyLogging {
+
+  val application = "SupportWorkers"
+
+  //Common Dimensions
+  val commonDimensions: Seq[Dimension] =
+    Seq(new Dimension().withName("Stage").withValue(Configuration.stage))
+
+  val allDimensions: Seq[Dimension] = commonDimensions ++ metrics
+
+  trait LoggingAsyncHandler extends AsyncHandler[PutMetricDataRequest, PutMetricDataResult] {
+    def onError(exception: Exception) {
+      logger.info(s"CloudWatch PutMetricDataRequest error: ${exception.getMessage}}")
+    }
+    def onSuccess(request: PutMetricDataRequest, result: PutMetricDataResult) {
```

Comment:
If this and `onError` are overriding or implementing a method in AsyncHandler use the `override` keyword to make this obvious.